accelerator-home-ui/dist/js/appBundle.js

```diff
@@ -66374,18 +66398,23 @@ preferredAudioLanguages:   preferredAudioLanguages$1
       console.log("App listenToVoiceControl method got called, configuring VoiceControl Plugin");
       await voiceApi.activate().then(() => {
         voiceApi.voiceStatus().then(voiceStatusResp => {
-          if (voiceStatusResp.ptt.status != "ready" || !voiceStatusResp.urlPtt.includes("avs://")) {
-            console.error("App voiceStatus says PTT/AVS not ready, enabling it.");
-            // TODO: Future -> add option for user to select which Voice service provider.
-            // Then configure VoiceControl plugin for that end point.
-            // TODO: voiceApi.configureVoice()
-            if (AlexaApi.get().checkAlexaAuthStatus() != "AlexaUserDenied") {
-              AlexaApi.get().setAlexaAuthStatus("");
-              voiceApi.configureVoice({
-                "enable": true
-              }).then(() => {
-                AlexaApi.get().setAlexaAuthStatus("AlexaAuthPending");
-              });
+          if (voiceStatusResp.success) {
+            if (voiceStatusResp.ptt.status != "ready" || !voiceStatusResp.urlPtt.includes("avs://")) {
+              GLOBALS.AlexaAvsstatus(false);
+              console.error("App voiceStatus says PTT/AVS not ready, enabling it.");
+              // TODO: Future -> add option for user to select which Voice service provider.
+              // Then configure VoiceControl plugin for that end point.
+              // TODO: voiceApi.configureVoice()
+              if (AlexaApi.get().checkAlexaAuthStatus() != "AlexaUserDenied") {
+                AlexaApi.get().setAlexaAuthStatus("");
+                voiceApi.configureVoice({
+                  "enable": true
+                }).then(() => {
+                  AlexaApi.get().setAlexaAuthStatus("AlexaAuthPending");
+                });
+              }
+            } else {
+              GLOBALS.AlexaAvsstatus(true);
```

Comment:
This logic seems wrong;
what will be if ptt.status is ready but urlPtt does not have "avs" in it ?
if (voiceStatusResp.ptt.status != "ready" || !voiceStatusResp.urlPtt.includes("avs://")) {